typescript/commands/build.py

```diff
@@ -0,0 +1,40 @@
+import sublime_plugin
+import sublime
+import os
+from ..libs.global_vars import IS_ST2
+
+
+class TypescriptBuildCommand(sublime_plugin.WindowCommand):
+    def run(self):
+        file_name = self.window.active_view().file_name()
+        directory = os.path.dirname(file_name)
+        if "tsconfig.json" in os.listdir(directory):
```

Comment:
There is no existing command on the server side to provide project info. However, I do think we need to create one for not only this and future features like project-wide error listing etc. I'll send a PR to the typescript repo soon